### DISCUSSION TITLE
HELP😭 在dataraw文件夹中放多个不同的speaker文件夹的问题-put various speaker folder under dataraw file

### DISCUSSION THREAD

**Discussion by popcorn333** [svc-develop-team/so-vits-svc#350] [General]
如果我在在dataraw文件夹中放多个不同的speaker文件夹， 那么训练出的结果是多个不同的speaker分别的模型还是融合了·所有speaker音色的单个模型？

If I put multiple different speaker folders in the dataraw folder, will the training result be separate models for multiple different speakers or a single model that combines all speaker timbres?

**Comment by bfloat16** [comment]
具体来说是一个模型学会了多个speaker的声音，推理的时候也是输出某一个speaker的声音

**Reply by popcorn333** [reply]
好哦 谢谢大佬。 我就是看contentVec 论文的时候 看到会有一个融合了·所有speaker音色 的stage 就迷糊了
